App.Loans/Controllers/LoansController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using App.Loans.Interface;
+
+namespace App.Loans.Controllers
+{
+
+        [Route("api/loans")]
+        [ApiController]
+        public class LoansController : ControllerBase
+        {
+            readonly IAnotherService _anotherService;
+            readonly ILoanManger _loansManager;
+            public LoansController(
+                IAnotherService anotherService,
+                ILoanManger loansManager)
+            {
+                _anotherService = anotherService;
+                _loansManager = loansManager;
+            }
+
+            // GET api/loans/values
+            [HttpGet("values")]
+            public ActionResult<IEnumerable<string>> GetActiveLoans()
+            {
+                _anotherService.DoAnything();
+                var serviceCallResult = _loansManager.GetValues().ToList();
+                return serviceCallResult;
+            }
+
+        // GET api/loans/{Id}
+        [HttpGet("{Id}")]
+        public ActionResult<IEnumerable<string>> GetAmountOfPaymentsLeft(int Id)
```

Comment:
The route for this endpoint should be like "api/loans/{id}/paymentsleft"
The route "api/loans/{Id}" should just return the loan object by it`s Id